packages/blockchain-wallet-v4-frontend/src/scenes/Transactions/RiskInvestment/Lines.styles.ts

```diff
@@ -0,0 +1,17 @@
+import styled, { css } from 'styled-components'
+
+export const ListContainer = styled.div<{
+  isCollapsed: boolean
+}>`
+  overflow: hidden;
+  transition: max-height 0.3s ease;
+
+  ${({ isCollapsed }) =>
+    isCollapsed
+      ? css`
+          max-height: 40px;
+        `
+      : css`
+          max-height: 200px;
+        `}
+`
```

Comment:
A personal opinion here - why not `max-height: ${(props) => props.isCollapsed ? '40px' : '200px'}` ?